src/Vortice.XInput/XInput13.cs

```diff
@@ -43,7 +43,7 @@ int IXInput.XInputGetAudioDeviceIds(int dwUserIndex, IntPtr renderDeviceId, IntP
             throw new NotSupportedException("XInputGetAudioDeviceIds is not supported on XInput1.3");
         }
 
-        [DllImport("xinput1_3.dll", CallingConvention = CallingConvention.StdCall)]
+        [DllImport("xinput1_3.dll", EntryPoint = "#100", CallingConvention = CallingConvention.StdCall)]
```

Comment:
Why this change?

Reply:
The regular entrypoint doesn't pass along the Guide flag, this one does.

Reply:
Would you prefer a property on XInput, e.g. "AllowUnofficialApi" that would toggle the use of these?